sickbeard/classes.py

```diff
@@ -212,7 +212,7 @@ def selectSeries(allSeries):
 
 
 class Proper(object):
-    def __init__(self, name, url, date, show, seeders, leechers, size, pubdate, hash, proper_tags):
+    def __init__(self, name, url, date, show, seeders, leechers, size, pubdate, torrent_hash, proper_tags):
```

Comment:
I think there both confusing without the right context.  I've been changing hash to torrent_hash pretty much everywhere else.